source/indexes/create-search-index.txt

```diff
@@ -0,0 +1,67 @@
+.. _compass-create-search-index:
+
+============================
+Create an Atlas Search Index
+============================
+
+You can create Atlas Search indexes in |compass|. Atlas Search indexes
+let you query data in `Atlas Search
+<https://www.mongodb.com/docs/atlas/atlas-search/>`__. Atlas Search
+indexes enable performant text search queries by mapping search terms to
+the documents that contain those terms.
+
+About this Task
+---------------
+
+To create an Atlas Search index, your deployment must be hosted on
+`MongoDB Atlas <https://www.mongodb.com/docs/atlas/>`__ and have a
+cluster tier of M10 or higher.
+
+Steps
+-----
+
+.. procedure::
+   :style: normal
+   
+   .. step:: Open the index creation dialog
+
+      From the :ref:`Indexes <collection-tab>` tab, click the
+      :guilabel:`Create` button, then click :guilabel:`Search Index`.
+
+   .. step:: Enter the index name
+
+      Specify a name for the index.
+
+   .. step:: Specify the search index definition
+
+      |compass-short| provides templates for different kinds of search
+      indexes. To learn more about search index definition syntax, see
+      :ref:`search-index-definition-create-mongosh`.
+
+   .. step:: Click :guilabel:`Create Search Index`
```

Comment:
**Suggestion**

I wonder if we could add some general guidance here for the general creation time / query considerations for when the Atlas Search Index is being created? [In the Atlas Docs ](https://www.mongodb.com/docs/atlas/atlas-search/create-index/#check-the-status)it simply says:

```
.. note::

   Larger collections take longer to index.
```

Reply:
This behavior isn't specific to Atlas Search indexes, so I think it's ok to not mention here. Generally we don't dive too deep into the details behind indexes in the Compass docs.

Reply:
SGTM resolved.